scss/_root.scss

```diff
@@ -158,6 +158,8 @@
   --#{$boosted-prefix}kbd-color: #{$kbd-color-inverted};
   --#{$boosted-prefix}kbd-bg: #{$kbd-bg-inverted};
   --#{$boosted-prefix}pre-color: #{$pre-color-inverted};
+  --#{$prefix}body-color: #{$body-color-dark};
+  --#{$prefix}heading-color: #{$white};
```

Comment:
Not a big fan to use `$white` (and `$black` after) rather than `$headings-color-*`. But I suppose it'll be tackled with the dark mode after in a more generic way?

Reply:
Need to check but I'd say the behavior must be consistent with the actual implementation of body text color in Boosted.

Reply:
I think it is better this way...